HomeEdu/Scenes/Main/ViewScore/ScoreViewController.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  ScoreViewController.swift
+//  HomeEdu
+//
+//  Created by Nguyen Cuong on 8/7/19.
+//  Copyright © 2019 Nguyen Cuong. All rights reserved.
+//
+
+import UIKit
+
+class ScoreViewController: UIViewController {
+
+    @IBOutlet weak var viewEachTerm: UIView!
+    @IBOutlet weak var viewAllScore: UIView!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+    
+    @IBAction func switchViewScore(_ sender: UISegmentedControl) {
+        if sender.selectedSegmentIndex == 0 {
+            viewEachTerm.alpha = 0
```

Comment:
```suggestion
            viewEachTerm.alpha = sender.selectedSegmentIndex == 0 ? 0 : 1
```
tương tự với `viewAllScore.alpha`